services/mailservice/mail_test.go

```diff
@@ -177,7 +177,7 @@ func TestSendMailUsingConfigAdvanced(t *testing.T) {
 	headers := make(map[string]string)
 	headers["TestHeader"] = "TestValue"
 
-	if err := SendMailUsingConfigAdvanced(mimeTo, smtpTo, from, emailSubject, emailBody, attachments, headers, cfg, true); err != nil {
+	if err := SendMailUsingConfigAdvanced(mimeTo, smtpTo, from, from, emailSubject, emailBody, attachments, headers, cfg, true); err != nil {
```

Comment:
For clarity, I would define a `replyTo` not to send the `from` twice, so with the rest of the variables at the top, I would add:

```
var replyTo = mail.Address{Name: "ReplyTo", Address: "reply_to@mattermost.com"}
```

and here:

```suggestion
	if err := SendMailUsingConfigAdvanced(mimeTo, smtpTo, from, replyTo, emailSubject, emailBody, attachments, headers, cfg, true); err != nil {
```